mcs/class/Makefile

```diff
@@ -50,6 +50,7 @@ endif
 
 mobile_common_dirs := \
 	corlib	\
+	Mono.Profiler.Log	\
```

Comment:
Mobile has limited available memory and this library load everything into memory so it does not seems to me as useful for mobile with current design.